react/ProductAssemblyOptions.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useMemo, FC, Fragment } from 'react'
+import useProduct from 'vtex.product-context/useProduct'
+import { find, propEq, compose, last, split, pathOr } from 'ramda'
+import ProductAssemblyOptionsGroupStateManager from './components/ProductAssemblyOptions/ProductAssemblyOptionsGroupStateManager'
```

Comment:
⛏ i think you could avoid this long-ass name since it's already under `ProductAssemblyOptions` thus the context is already clear